program_info/CMakeLists.txt

```diff
@@ -38,6 +38,14 @@ set(Launcher_Branding_LogoQRC "program_info/prismlauncher.qrc" PARENT_SCOPE)
 
 set(Launcher_Portable_File "program_info/portable.txt" PARENT_SCOPE)
 
+if(CMAKE_BUILD_TYPE STREQUAL "Debug" AND NOT MACOSX_PREPARE_NOTARIZATION)
+    set(Launcher_DebugEntitlements "<key>com.apple.security.get-task-allow</key>\n    <true/>")
+else()
+    set(Launcher_DebugEntitlements "")
+endif()
+
+configure_file(App.entitlements App.entitlements)
```

Comment:
if we're going to configure the file via cmake it should be renamed to a `*.in` sufix

Reply:
Makes sense, will do that

I'm also realizing looking at this again I should probably simply configure the ad-hoc entitlement into the file rather than having a separate file for it if I'm already configuring for the debug entitlement with cmake. I'll do those together soon.